packages/gasket-plugin-swagger/index.js

```diff
@@ -89,6 +99,10 @@ module.exports = {
 
       const swaggerSpec = await loadSwaggerSpec(root, definitionFile);
 
+      if (!swaggerSpec) {
+        gasket.logger.error('Missing swagger.json file...');
```

Comment:
User and change the `definitionFile` to be something other than `swagger.json` so use that in your warning.
```suggestion
        gasket.logger.error(`Missing ${definitionFile} file...`);
```

Also, it might be better to move this logging into the `loadSwaggerSpec`. You can pass the logger as an argument. That'll keep the logging paired with the loading code.